lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc

```diff
@@ -278,7 +279,7 @@ function dosomething_reportback_views_default_views() {
   $handler->display->display_options['menu']['context_only_inline'] = 0;
 
   /* Display: Page */
-  $handler = $view->new_display('page', 'Page', 'page_1');
+  $handler = $view->new_display('page', 'Page', 'reportback_review');
```

Comment:
this should be the page of already reviewed reportbacks, not the page to review...right?